client/utils/test/__snapshots__/account-fees.tsx.snap

```diff
@@ -0,0 +1,502 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Account fees utility functions formatMethodFeesTooltip() displays base fee, when only promo discount without percentage or fixed 1`] = `
+Object {
```

Comment:
I think is better to have snapshot of the rendered HTML, please look at my previous comment for more contet.

Reply:
Thanks. Made the changes here ca527b5 

I can see the snapshot now showing only the HTML  ( `baseElement` property of the rendered object ) . I hope this is correct.

Reply:
Thanks for all the time and details you gave me here! I learned a lot! 🙌